AsyncDisplayKit/Details/Transactions/_ASAsyncTransaction.mm

```diff
@@ -508,7 +508,7 @@ - (void)_ensureTransactionData
 
 - (NSString *)description
 {
-  return [NSString stringWithFormat:@"<_ASAsyncTransaction: %p - _state = %lu, _group = %p, _operations = %@>", self, (unsigned long)_state, _group, _operations];
+  return [NSString stringWithFormat:@"<_ASAsyncTransaction: %p - _state = %lu, _group = %p, _operations = %@>", self, (unsigned long)__atomic_load_n(&_state, __ATOMIC_SEQ_CST), _group, _operations];
```

Comment:
I honestly don't know the code well enough to give you an authoritative answer.  In all honesty, the `__atomic_...` changes regarding the `_state` ivar were done mostly to silence the warnings / errors being reported by Thread Sanitizer.

So, to be clear, there may be an actual problem here that needs to be addressed instead of just silenced.  IMHO, this should be reviewed by someone who has a thorough understanding of the code base and deep knowledge of concurrency issues.